Automattic/wp-calypso
# Add Emails to fallback nav menu

#### Changes proposed in this Pull Request

* This PR adds the new Emails item to the Upgrades submenu conditional on the `email/centralized-home` feature flag being enabled.
* This PR also re-orders the items in that menu to match the sequence we want as per #51150.

#### Testing instructions

* Open this change in the [live branch](https://calypso.live/?branch=add/emails-menu-to-calypso-fallback), and navigate to `/email/:siteSlug?flags=-centralized-home`.
* Use the browser console to ensure that the local Calypso Indexed DB data has been cleared.
* Refresh the page and verify that the left nav does not the Upgrades menu expanded, and doesn't have the "Emails" submenu highlighted.
* Remove the Indexed DB data for Calypso again, and immediately refresh the page but with `?flags=email/centralized-home` as the suffix (to enable the feature)
* Verify that you briefly see the Upgrades menu expanded and the Emails submenu highlighted before being updated/removed by the response from the server
    - This may need to be retried to fully verify the behaviour, as the fallback logic only triggers before we get the admin-menu response back from the server.

##### Screenshot

<img width="276" alt="Screenshot 2021-04-26 at 21 53 56" src="https://user-images.githubusercontent.com/3376401/116142252-f85eab80-a6d9-11eb-88d9-fe050a163ea2.png">

Related to #51150
